project/hooks/pre-commit

```diff
@@ -9,19 +9,19 @@ fi
 
 cd $REPO_ROOT
 
-GOFMT_ERRORS=$(gofmt -s -l . 2>&1)
+GOFMT_ERRORS=$(gofmt -s -l $(find . -type f -name '*.go' -not -path "./vendor/*") 2>&1)
```

Comment:
I would prefer `*/vendor/*`, just had this issue when adding a package with its own vendor in contrib